src/pyedb/dotnet/edb_core/edb_data/control_file.py

```diff
@@ -1262,6 +1265,42 @@ def parse_xml(self, xml_input):
                                                 via.remove_unconnected = (
                                                     True if i.attrib["RemoveUnconnected"] == "true" else False
                                                 )
+            if el.tag == "Boundaries":
+                for port_el in el:
+                    if port_el.tag == "CircuitPortPt":
+                        self.boundaries.add_port(
+                            name=port_el.attrib["Name"],
+                            x1=port_el.attrib["x1"],
+                            y1=port_el.attrib["y1"],
+                            layer1=port_el.attrib["Layer1"],
+                            x2=port_el.attrib["x2"],
+                            y2=port_el.attrib["y2"],
+                            layer2=port_el.attrib["Layer2"],
+                            z0=port_el.attrib["Z0"],
+                        )
+            if el.tag == "SimulationSetups":
```

Comment:
Can you use find from ElementTree instead of these nested for loops ? 
This makes this hard to read and maintain